src/addrdb.cpp

```diff
@@ -213,9 +214,11 @@ void DumpAnchors(const fs::path& anchors_db_path, const std::vector<CAddress>& a
 std::vector<CAddress> ReadAnchors(const fs::path& anchors_db_path)
 {
     std::vector<CAddress> anchors;
+    int64_t nStart = GetTimeMillis();
+
     try {
         DeserializeFileDB(anchors_db_path, anchors, CLIENT_VERSION | ADDRV2_FORMAT);
-        LogPrintf("Loaded %i addresses from %s\n", anchors.size(), fs::quoted(fs::PathToString(anchors_db_path.filename())));
+        LogPrintf("Loaded %i addresses from %s %dms\n", anchors.size(), fs::PathToString(anchors_db_path.filename()), GetTimeMillis() - nStart);
```

Comment:
I'm not sure about moving away from fs::quoted here. There's also other places where quoted is used for logging. I'm not sure what's the plan there, but I'd like to avoid back-and-forth.

Reply:
I agree.. I will close it.